Xamarin.PropertyEditing/ViewModels/PropertiesViewModel.cs

```diff
@@ -164,6 +204,38 @@ private PropertyViewModel GetViewModel (IPropertyInfo property)
 			return new StringPropertyViewModel (property, this.editors);
 		}
 
+		void FilterData ()
+		{
+			this.properties.Clear ();
+
+			UpdateProperties ();
```

Comment:
I've tried various ways of caching and they just aren't working. It may be how NS*View populates. The following works, but obviously doesn't in cached. 

`public IReadOnlyList<PropertyViewModel> Properties { 
			get {
				if (!IsFiltering) {
					return this.properties;
				}
				else {
					return this.properties.Where ((arg) => arg.Property.Name.StartsWith (filterText, StringComparison.InvariantCultureIgnoreCase)).ToList ();
				}
			}
		}`